README.md

```diff
@@ -197,6 +197,19 @@ suspend fun main() {
 
 A mapping of [Discord's Voice Connection](https://discord.com/developers/docs/topics/voice-connections), which maintains the connection and handles audio transmission.
 
+If you want to use voice, you need to enable the voice capability,
+which is only available for Gradle
+
+```kotlin
+dependencies {
+    implementation("dev.kord", "core", "<version>") {
+        capabilities {
+            requireCapability("dev.kord:core-voice:<version>")
```

Comment:
core-voice?
thought it was just voice

Reply:
This is a variant of core, including voice, the normal voice module remains